alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php

```diff
@@ -99,7 +99,7 @@ protected function getPort ($dc)
 	{
 		$port = kBroadcastUrlManager::DEFAULT_PORT;
 	
-		$broadcastConfig = $this->getConfiguration();	
+		$broadcastConfig = $this->getConfiguration($dc);	
```

Comment:
The configuration support port in the top level, once for all DCs, don't enforce here the dc level.